<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-IJVEGUQ" failures="1" tests="2" name="Default test(failed)" time="0.149" errors="0" timestamp="15 Jan 2025 08:22:15 GMT">
  <testcase classname="com.re_run_testcase.TestNgRetryFailedDemo" name="test03" time="0.063">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.re_run_testcase.TestNgRetryFailedDemo.test03(TestNgRetryFailedDemo.java:22)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test03 -->
  <testcase classname="com.re_run_testcase.TestNgRetryFailedDemo" name="test02" time="0.025"/>
  <testcase classname="com.re_run_testcase.TestNgRetryFailedDemo" name="test01" time="0.0">
    <ignored/>
  </testcase> <!-- test01 -->
</testsuite> <!-- Default test(failed) -->
